src/status_im2/contexts/activity_center/view.cljs

```diff
@@ -201,24 +203,27 @@
            :else
            nil)]))))
 
-(defn view
-  [request-close]
+(defn f-view
```

Comment:
why are we prepending with `f-` here? 🤔

Reply:
because this function must be used with `:f>` i don't like it , waiting for better suggestions :)